var/spack/repos/builtin/packages/mofem-fracture-module/package.py

```diff
@@ -0,0 +1,107 @@
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+
+class MofemFractureModule(CMakePackage):
+    """mofem fracture module"""
+
+    homepage = "http://mofem.eng.gla.ac.uk"
+    url = "https://bitbucket.org/likask/mofem_um_fracture_mechanics"
+
+    maintainers = ['likask']
+
+    version('0.9.42',
+        git='https://bitbucket.org/likask/mofem_um_fracture_mechanics',
+        tag='v0.9.42')
+    version('develop',
+        git='https://bitbucket.org/likask/mofem_um_fracture_mechanics',
+        branch='develop')
+
+    variant('copy_user_modules', default=True,
+        description='Copy user modules directory instead linking')
+
+    extends('mofem-cephas')
+    depends_on("mofem-users-modules", type=('build', 'link', 'run'))
+
+    # The CMakeLists.txt installed with mofem-cephas package set cmake
+    # environment to install extension from extension repository. It searches
+    # for modules in user provides paths, for example in Spack source path.Also
+    # it finds all cmake exported targets installed in lib directory, which are
+    # built with dependent extensions, f.e.mofem - users - modules or others if
+    # needed.
+    @property
+    def root_cmakelists_dir(self):
+        """The relative path to the directory containing CMakeLists.txt
+
+        This path is relative to the root of the extracted tarball,
+        not to the ``build_directory``. Defaults to the current directory.
+
+        :return: directory containing CMakeLists.txt
+        """
+        spec = self.spec
+        return spec['mofem-cephas'].prefix.users_modules
+
+    def cmake_args(self):
+        spec = self.spec
+        source = self.stage.source_path
+
+        options = []
+
+        # obligatory options
+        options.extend([
+            '-DWITH_SPACK=YES',
+            '-DEXTERNAL_MODULES_BUILD=YES',
+            '-DUM_INSTALL_BREFIX=%s' % spec['mofem-users-modules'].prefix,
+            '-DEXTERNAL_MODULE_SOURCE_DIRS=%s' % source,
+            '-DSTAND_ALLONE_USERS_MODULES=%s' %
```

Comment:
I noticed in some work related to cmake package updates: should this be `ALONE` not `ALLONE`?